tests/test_parfile_writing_format.py

```diff
@@ -95,6 +95,14 @@ def test_ECL():
             assert l.split()[-1] == "IERS2003"
 
 
+def test_DMDATA():
+    """Should be an integer for TEMPO/TEMPO2"""
+    m = get_model(os.path.join(datadir, "J0030+0451_post.par"))
+    for l in m.as_parfile(format="tempo").split("\n"):
+        if "DMDATA" in l:
+            assert isinstance(int(l.split()[-1]), int)
```

Comment:
I don't think there's ever a situation where the function `int` returns a non-integer; should this test just be to call `int` and see if it raises an exception? How does this handle `0.0` and the like - floating-point numbers that are integers? I assume TEMPO/TEMPO2 don't like those?